terraform/modules/front_door/outputs.tf

```diff
@@ -9,7 +9,7 @@ output "load_balancer_arn_suffix" {
 }
 
 output "load_balancer_listener_arn" {
-  value       = aws_lb_listener.https[0].arn
+  value       = var.initial_create ? null : aws_lb_listener.https[0].arn

```

Comment:
When initially creating infra, this output errored so I added this conditional

Reply:
Huh, I think I just saw it as blank before rather than erroring. 🤷‍♀️